v2/component/launcher.go

```diff
@@ -712,8 +716,14 @@ func getExecutorOutput() (*pipeline_spec.ExecutorOutput, error) {
 
 func (l *Launcher) openBucket() (*blob.Bucket, error) {
 	if l.bucketConfig.scheme == "minio://" {
+		secret, err := getMinioCredential()
+		if err != nil {
+			return nil, fmt.Errorf("Failed to get minio credential: %v", err)
```

Comment:
```suggestion
			return nil, fmt.Errorf("Failed to get minio credential: %w", err)
```

use `%w` to wrap errors?

Similarly for other cases, I think we typically prefer `%w` over '%v' for errors.